OpenShift/nexus-repository-manager.yaml

```diff
@@ -1,159 +0,0 @@
-apiVersion: v1
```

Comment:
do these deleted files go away because they're built from smaller components now? or what were they? I guess they're not even in the helm directory.

Reply:
I'm not sure what this file is/was